src/shell/job.rs

```diff
@@ -37,10 +37,10 @@ impl Job {
             let mut iterator = self.args.drain();
             expanded.push(iterator.next().unwrap());
             for arg in iterator.flat_map(|argument| expand_string(&argument, expanders, false)) {
-                if arg.contains(|chr| chr == '?' || chr == '*' || chr == '[') {
+                /*if arg.contains(|chr| chr == '?' || chr == '*' || chr == '[') {
```

Comment:
Perhaps also add a test in the **examples** directory that checks if the following functions correctly? All you do is create the ion file that contains the commands to run, and the corresponding out file that will have the correct output of the standard output of the ion script when execute.

```
echo br*.*
> braces.ion braces.out break.ion break.out
echo [ef]*.ion
> else_if.ion else_if.out fail.ion fail.out fibonacci.ion fibonacci.out fn.ion fn.out for.ion for.out
```